miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -594,18 +594,12 @@ HotTarget.sample = (function() {
               }
 
               var classes = getSampleClasses(samples);
-              var categories = Utils.array.deduplicateString(classes.map(function(sampleClass) {
-                return sampleClass.sampleCategory;
+              var classesAliases = Utils.array.deduplicateString(classes.map(function(sampleClass) {
+                return sampleClass.alias;
               }));
-              if (categories.length > 1) {
-                alert("You have selected samples of categories " + categories.join(" & ")
-                    + ". Please select samples from only one category.");
-                return;
-              }
-
-              if (categories[0] == 'Tissue Processing' && classes.length > 1) {
-                alert("You have selected samples of classes " + classes.map(Utils.array.getAlias).join(" & ")
-                    + ". Please select samples from only one tissue processing class.");
+              if (classesAliases.length > 1) {
+                alert("You have selected samples of classes " + classesAliases.join(" & ")
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this usage of alert(...). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)